src/styles/helpers/__test__/replaceAlpha.spec.js

```diff
@@ -0,0 +1,26 @@
+import { replaceAlpha } from '../color';
+
+describe('replaceAlpha', () => {
+    const alpha = 0.5;
+    const colors = [
+        {
+            rgb: 'rgb(1, 182, 245)',
+            rgba: 'rgba(1, 182, 245, 0.3)',
+            rgbar: `rgba(1, 182, 245, ${alpha})`,
+        },
+    ];
```

Comment:
Since here you are passing an array with one element, use it directly instead of use forEach.